boa/src/builtins/map/mod.rs

```diff
@@ -46,6 +48,11 @@ impl BuiltIn for Map {
                 entries_function.clone(),
                 Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE,
             )
+            .property(
+                to_string_tag,
+                "Math",
```

Comment:
Shouldn't this be `Map`?
```suggestion
                "Map",
```

Reply:
Yep, my bad.